scripts/apps/authoring/authoring/directives/AuthoringTopbarDirective.ts

```diff
@@ -26,6 +26,7 @@ export function AuthoringTopbarDirective(TranslationService, privileges, authori
             scope.saveTopbar = function() {
                 scope.saveDisabled = true;
                 return scope.save(scope.item)
+                    .then((item) => angular.extend(scope.item, item))
```

Comment:
why is this needed here?

Reply:
to be sure that after save the item has the same value as origItem

Reply:
shouldn't it be part of that `scope.save` method then?

Reply:
OK, moved on save